projects/core/src/lib/datatable/table/table.component.scss

```diff
@@ -15,6 +18,20 @@ table {
     ::ng-deep tbody tr {
         height: 2rem !important;
     }
+
+    th,
+    td {
+        white-space: nowrap;
+        padding: 1rem;
```

Comment:
I think you create too much vertical padding to reflect a material design table.

![grafik](https://user-images.githubusercontent.com/4947061/189663036-8f1d0714-44e7-4c18-a30b-dbf06e8d6ce4.png)

Here, it seems to be 48 px (3rem) of height.

We have now something like 74.8px of vertical space.

The horizontal space seems fine, although maybe 1rem instead of 2rem of padding would be better to have a more dense table.

It would look more like this:
![grafik](https://user-images.githubusercontent.com/4947061/189663862-5e6ac55b-e798-4eff-8cd6-2559fb585a45.png)

Reply:
6559f80